android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json

```diff
@@ -0,0 +1,122 @@
+{
+  "appId": "app",
+  "configType": "profile",
+  "id": "inventoryProfile",
+  "fhirResource": {
+    "baseResource": {
+      "resource": "Group"
+    },
+    "relatedResources": [
+      {
+        "resource": "Observation",
+        "searchParameter": "subject"
+      }
+    ]
+  },
+  "rules": [
+    {
+      "name": "commodityName",
+      "condition": "true",
+      "actions": [
+        "data.put('commodityName', fhirPath.extractValue(Group, 'Group.name'))"
+      ]
+    }
+  ],
+  "views": [
+    {
+      "viewType": "COLUMN",
+      "children": [
+        {
+          "viewType": "CARD",
+          "padding": 0,
+          "content": [
+            {
+              "viewType": "COLUMN",
+              "children": [
+                {
+                  "viewType": "COMPOUND_TEXT",
+                  "primaryText": "@{commodityName}",
+                  "primaryTextColor": "#000000"
+                },
+                {
+                  "viewType": "SPACER",
+                  "height": 10
+                },
+                {
+                  "viewType": "ROW",
+                  "children": [
+                    {
+                      "viewType": "PERSONAL_DATA",
+                      "personalDataItems": [
+                        {
+                          "label": {
+                            "viewType": "COMPOUND_TEXT",
+                            "primaryText": "82",
+                            "primaryTextColor": "#000000"
+                          },
+                          "displayValue": {
+                            "viewType": "COMPOUND_TEXT",
+                            "primaryText": "Available",
```

Comment:
Should this be computed?

Reply:
Nope, we will update them once we have finalized the rules.